src/Microsoft.Framework.Runtime.Interfaces/Compilation/ResourceDescriptor.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Framework.Runtime.Compilation
 {
     public class ResourceDescriptor
     {
-        public string Name { get; set; }
+        public string Path { get; set; }
```

Comment:
Will change back to `name`, it is a left over from a previous attempt that I forgot to revert